Userland/Utilities/test-pthread.cpp

```diff
@@ -31,8 +31,11 @@ static ErrorOr<void> test_once()
         })));
         threads.last().start();
     }
+    // clang-format off
+    // It wants to put [[maybe_unused]] on its own line, for some reason.
```

Comment:
I've opened https://github.com/llvm/llvm-project/issues/59235 for this

Reply:
Should have tested more.
Basically it seems that annotations following these control flow statements without braces are somewhat ambiguous in C++, and it only works here because of the declaration (`auto res`).
But because of the declaration, I don't even think maybe_unused is strictly required *by the compiler* (which doesn't mean it shouldn't be there at all to communicate something to the reader)